test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1607,34 +1607,44 @@ def test_embedding_conv_linear_quantization(self):
         )
 
     def test_move_exported_model_to_eval(self):
-        class M(torch.nn.Module):
-            def __init__(self):
-                super().__init__()
-                self.dropout = torch.nn.Dropout(0.5)
-
-            def forward(self, x):
-                return self.dropout(x)
-
-        example_inputs = (torch.randn(1),)
-        m = M().train()
-        m = capture_pre_autograd_graph(m, example_inputs)
 
-        # Assert that dropout op exists and is in train mode
-        dropout_node = None
-        for n in m.graph.nodes:
-            if n.target == torch.ops.aten.native_dropout.default:
-                dropout_node = n
-                break
-        self.assertTrue(dropout_node is not None)
-        self.assertTrue(dropout_node.args[2])
-
-        # Do the subgraph rewriting
-        torch.ao.quantization.move_exported_model_to_eval(m)
-
-        # Assert that dropout op is now replaced with a clone op
-        targets = [n.target for n in m.graph.nodes]
-        self.assertTrue(torch.ops.aten.clone.default in targets)
-        self.assertTrue(torch.ops.aten.native_dropout.default not in targets)
+        for inplace in [True, False]:
```

Comment:
Do you mind refactoring this into a helper function so we have something like:

```
def test_move_exported_model_to_eval(self):
    self._test_move_exported_model_to_eval_dropout(inplace=True)
    self._test_move_exported_model_to_eval_dropout(inplace=False)
```
This will make it easier for us to tell which case failed if things break in the future

Reply:
Thanks for the suggestions and modified.